Doctrine/Orm/Extension/PaginationExtension.php

```diff
@@ -93,12 +93,16 @@ public function getResult(QueryBuilder $queryBuilder)
      * Checks if the pagination is enabled or not.
      *
      * @param ResourceInterface $resource
-     * @param Request           $request
+     * @param Request|null      $request
      *
      * @return bool
      */
-    private function isPaginationEnabled(ResourceInterface $resource, Request $request)
+    private function isPaginationEnabled(ResourceInterface $resource, Request $request = null)
```

Comment:
`isPaginationEnabled` shouldn't have to handle cases of `$request` being `null`. Separation of concerns...